c2corg_ui/static/partials/map/map.html

```diff
@@ -8,7 +8,11 @@
     class="btn btn-primary map-reset-feature" translate>Reset feature</button>
   <div class="map-filter-switch" ng-show="::mapCtrl.advancedSearch">
     <label>
-      <input type="checkbox" ng-model="mapCtrl.enableMapFilter"><span translate>Filter on map extent</span>
+      <input type="checkbox" ng-model="mapCtrl.enableMapFilter">
+      <div>
+        <span translate class="text">Filter on map extent</span>
+        <span class="glyphicon glyphicon-globe"></span>
```

Comment:
Are you sure this really works? I am testing this branch in my instance: no "glyphicon-globe" appears!? By the way I am not really convinced that we should display such an icon, since we don't have that much space when the map is small.